### ISSUE TITLE
Max HP Modifier updates the Max HP insead of than Temp Max HP

### ISSUE THREAD

**Issue by Janders797** [kakaroto/Beyond20#1158]
When HP are temporarily modified on DNDBeyond (Aid, Specter, etc), this should change the Temp Max HP, not adjust the Maximum Hitpoints.
DBDBeyond:
![image](https://github.com/user-attachments/assets/3201b633-9670-4557-92cf-aa685a92abf9)
Foundry:
![image](https://github.com/user-attachments/assets/5c98cf92-3d3f-422c-85f3-9fe10e887b7d)

**Comment by kakaroto** [comment]
That's incorrect, temp HP is not the same as HP. You can literally see in your first screenshot how "Temp HP" is marked as 0 in D&D Beyond.
If you get hit, you lose the temp HP but cannot regain it, while your hit point maximum being increased by Aid would allow you to heal back to the new maximum for the duration of the spell. Spectre monster would lower your hitpoint maximum, that doesn't make it negative temp HP either.
I think you misunderstand the concept of temp HP and how modifying a character's maximum HP works in D&D 5e.

**Comment by Janders797** [comment]
It shouldn't modify the Max HP, nor the Temp HP, but the box under it, called Temp Max HP. Which it clearly didn't.

**Comment by Janders797** [comment]
This is what it's supposed to look like:
![image](https://github.com/user-attachments/assets/e3888fdc-f57e-4f76-a383-57c537fd9265)

**Comment by kakaroto** [comment]
Ah, I see, thanks for the clarification, I misunderstood the original request, and I hadn't understood the "Temp Max HP" (understood it as "Max Temp HP" which doesn't make sense, lol). I think it was a recent addition to the dnd5e system (at least, it wasn't a thing when I implemented the original hp syncing feature.
